SConstruct

```diff
@@ -866,8 +866,9 @@ elif "clang" in env.subst("$CC"):
     config.select("clang")
 
 else:
-    logger.error(f"Unrecognized C compiler {env['CC']!r}")
-    sys.exit(1)
+    # Assume a GCC compatible compiler if nothing else
+    config.select("gcc")
```

Comment:
Will this make things harder to debug?

Reply:
Perhaps in some corner case, but I think it's somewhat unlikely. With the exception of MSVC, it seems that most compilers accept the same flags as GCC for the options we want to set. I changed this so there will at least be a warning that GCC variables were assumed.

There's only so we can do with our ad hoc, name-based identification of the compiler (and I'm not advocating making this more complicated), and I'd rather have a likely fallback in place than force the user to edit `SConstruct`. If this is wrong, at least the user can specify all the right flags through other SCons options.

Reply:
The logger warning is definitely appreciated. Thanks!